webpack.umd.config.js

```diff
@@ -13,6 +13,7 @@ const webpackUMDConfig = {
     [pkg.name]: paths.src('umd.js'),
   },
   externals: {
+    'prop-types': 'PropTypes',
```

Comment:
I believe per their instructions we should bundle propTypes with the code as a dependency.  Otherwise, users will also have to make it globally available.  I'll update this.

Just as a heads up, the Codepen is powered by the UMD build as well so any UMD changes also require updates there.